app/scripts/views/settings/clients.js

```diff
@@ -45,28 +49,45 @@ define(function (require, exports, module) {
       }
 
       var devices = this._devices;
-      devices.on('add', this._onDeviceAdded.bind(this));
-      devices.on('remove', this._onDeviceRemoved.bind(this));
+      devices.on('add', this._onItemAdded.bind(this));
+      devices.on('remove', this._onItemRemoved.bind(this));
+
+      if (! this._apps) {
+        this._apps = new Apps([], {
+          notifier: options.notifier
+        });
+      }
+
+      this._apps.on('add', this._onItemAdded.bind(this));
```

Comment:
Instead of creating two separate Collections and combining them into a third collection for rendering, could an AppsAndDevicesCollection be passed in that knows which of the two model types to add to itself?